src/components/forms/EventStreamsForm.js

```diff
@@ -0,0 +1,200 @@
+    let value = event.target.value.replace(/\s*/g, "");
+    if (value === "") value = null;
+    this.setState({ private_ip_allowlist: value });
+  }
+
+  /**
+   * handle input change
+   * @param {event} event event
+   */
+  handleInputChange(event) {
+    this.setState(this.eventTargetToNameAndValue(event));
+  }
+
+  /**
+   * Handle input change for a select
+   * @param {event} event
+   */
+  handlePlanChange(event) {
+    let item = event.target.value;
+    this.setState({ plan: item.toLowerCase() });
+  }
+
+  render() {
+    let composedId = `event-streams-form-${this.props.data.name}`;
+    return (
+      <>
+        <IcseFormGroup>
+          {/* name */}
+          <IcseNameInput
+            id={composedId}
+            componentName={"event_streams"}
+            value={this.state.name}
+            onChange={this.handleInputChange}
+            hideHelperText
+            invalid={this.props.invalidCallback(this.state, this.props)}
+            invalidText={this.props.invalidTextCallback(this.state, this.props)}
+          />
+          {/* plan */}
+          <IcseSelect
+            formName="Event Streams"
+            value={titleCase(this.state.plan)}
+            groups={this.props.plans}
+            handleInputChange={this.handlePlanChange}
+            className="fieldWidth"
+            name="plan"
+            labelText="Plan"
+          />
+          {/* resource group */}
+          <IcseSelect
+            formName="Event Streams"
+            value={this.state.resource_group}
+            groups={this.props.resourceGroups}
+            handleInputChange={this.handleInputChange}
+            className="fieldWidth"
+            name="resource_group"
+            labelText="Resource Group"
+          />
+        </IcseFormGroup>
+        {this.state.plan === "enterprise" && (
+          <>
+            <IcseFormGroup>
+              {/* endpoints */}
+              <EndpointSelect
+                formName="Event Streams"
+                handleInputChange={this.handleInputChange}
+                value={this.state.endpoint}
+                className="fieldWidth"
+              />
+              {/* throughput */}
+              <IcseSelect
+                formName="Event Streams"
+                value={
+                  this.state.plan !== "enterprise"
+                    ? throughputAndStorageOptions(this.state.plan).throughput[0]
+                    : this.state.throughput
+                }
+                groups={
+                  this.props.throughputs === null
+                    ? throughputAndStorageOptions(this.state.plan).throughput
+                    : this.props.throughputs
+                }
+                handleInputChange={this.handleInputChange}
+                className="fieldWidth"
+                name="throughput"
+                labelText="Throughput"
+              />
+              {/* storage size */}
+              <IcseSelect
```

Comment:
for these, hardcode the groups and then only display these two fields if plan === enterprise

Reply:
```js
            {this.state.plan === "enterprise"  && <IcseFormGroup>
                formName={this.props.data.name + " Event Streams"} 
                value={
                  // hardcode groups
                }
                groups={
                 // hardcode groups
                }
                handleInputChange={this.handleInputChange}
                className="fieldWidth"
                name="storage_size"
                labelText="Storage Size"
              />
            </IcseFormGroup>}